### DISCUSSION TITLE
Jyers Firmware on Ender-3 Max

### DISCUSSION THREAD

**Discussion by janchristophmartin** [Jyers/Marlin#1120] [Ideas]
Having happily used JyersUI Marlin for a while on my Ender-3 V2 inspired me to modding my Ender-3 Max with the V2 display unit and adapting the configuration to the Ender-3 Max. Initial tests seem to run fine, however I'd still consider this more as an experiment so far.

For anyone interested, please find the files and a short documentation here:
* Configurations: https://github.com/janchristophmartin/marlin-jyers-ender3max
* Wiki documentation: https://github.com/janchristophmartin/marlin-jyers-ender3max/wiki

I'd be glad to get some feedback should someone be willing to try out.

**Comment by smooreace** [comment]
Its just a matter of plugin in the v2 display. The rest its all the same,
no? The Max runs the 4.2.2 board if I'm not mistaken?

**Reply by janchristophmartin** [reply]
Indeed, swapping the display on the Ender-3 Max is quite straightforward. Most changes in the configuration refer to the larger printing volume and the different BLTouch sensor position.

I'm quite happy to see the Ender-3 Max with the more elegant V2 display together with all the functions the Jyers firmware offers.

**Comment by smooreace** [comment]
The Max is really a bad buy. The cost savings with the v2 can be applied to
the addition of the ender extender upgrade... leading to a max build area
of 400x400x500. thats where the shit gets real!

**Reply by janchristophmartin** [reply]
Maybe right, especially if you already have the Ender-3 V2 and want the 400x400x500 build volume without buying a complete new printer.

I wanted a second printer anyway, was fine with the 300x300 print bed and got a good deal on the Ender-3 Max. Plus I didn't want to go through the whole assembly process.

**Comment by anthonyonions** [comment]
I've just installed Jyers mesh bed firmware on my v2 and would also like to install it on my Max. However, I'm not needing to update the display as I tend to use octopi unless the printer is on the workbench for maintenance. 

Would this be usable without upgrading the display unit?

**Comment by smooreace** [comment]
The firmware wont work unless youre using the v2 display
